www/index.html

```diff
@@ -25,7 +25,7 @@
 
 <div id="platform-instructions-unix" class="instructions display-none">
   <p>Run the following in your terminal, then follow the onscreen instructions.</p>
-  <pre>curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh</pre>
+  <pre>curl --tlsv1.2 -sSf https://sh.rustup.rs | sh</pre>
```

Comment:
What about removing redundant https scheme: https://curl.haxx.se/docs/manpage.html#--proto
```suggestion
  <pre>curl --tlsv1.2 --proto '=https' -sSf sh.rustup.rs | sh</pre>
```

Reply:
This is not working:
`$ curl --tlsv1.2 --proto '=https' -sSf sh.rustup.rs`
`curl: (1) Protocol "http" not supported or disabled in libcurl`

Reply:
Oh, that's right.